health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/components/FormulaConfigScreen.js

```
@@ -0,0 +1,90 @@
+                                    output={ob.output}
+                                    input1={ob.input}
+                                    input2={ob.operator}
+                                    input3={ob.assumptionValue}
+                                />
+                            </>
+
+                        )
+                    }))}
+                </>
+            )
+        })
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)